Algorithms/GRAPHS/DFS &BFS/BFS&DFS.CPP

```diff
@@ -0,0 +1,75 @@
+#include<iostream.h>
```

Comment:
Graphs already has a Traversals folder in which DFS and BFS are already present. 
Please update your PR and delete your DFS/BFS.